app/services/coin_market_cap_pro/update_market_pairs_service.rb

```diff
@@ -50,7 +51,7 @@ def perform_update_pairs(identifier, data)
           :pair => pair["market_pair"],
           :price => quote["price"],
           :volume24h => volume24,
-          :volume_percentage => (volume24 / base_volume24 unless base_volume24.blank?),
+          :volume_percentage => (volume24 / base_volume24 if has_base_volume24),
```

Comment:
Maybe an if/else might be better here? Otherwise is `volume_percentage` going to be null?

i.e. `:volume_percentage => (if has_base_volume24 then volume24 / base_volume24 else 0 end),